src/devices/LidarLiteV3/LidarLiteV3.cs

```diff
@@ -0,0 +1,122 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers.Binary;
+using System.Collections.Generic;
+using System.Device;
+using System.Device.I2c;
+using System.IO;
+using System.Numerics;
+using System.Text;
+using System.Threading;
+
+namespace Iot.Device.LidarLiteV3
+{
+    /// <summary>
+    /// Lidar Lite v3 is a long-range fixed position distance sensor by Garmin.
+    /// </summary>
+    public class LidarLiteV3 : IDisposable
+    {
+        /// <summary>
+        /// Default address for LidarLiteV3
+        /// </summary>
+        public const byte DefaultI2cAddress = 0x62;
+
+        internal I2cDevice _i2cDevice;
+
+        /// <summary>
+        /// Initialize the LidarLiteV3
+        /// </summary>
+        /// <param name="i2cDevice">The I2C device</param>
+        public LidarLiteV3(I2cDevice i2cDevice)
+        {
+            _i2cDevice = i2cDevice;
+            Reset();            
+        }
+
+        /// <summary>
+        /// Reset FPGA, all registers return to default values
+        /// </summary>
+        public void Reset() 
+        {
+            try {
+                WriteRegister(Register.ACQ_COMMAND, 0x00);
+            } catch (System.IO.IOException)
+            {
+                // This exception is expected due to the reset.                
+            }
+        }
+
+        /// <summary>
+        /// Measure distance
+        /// </summary>
+        public ushort MeasureDistance(bool withReceiverBiasCorrection = true) 
+        {
+            if(withReceiverBiasCorrection) {
+                WriteRegister(Register.ACQ_COMMAND, 0x04);
+            } else {
+                WriteRegister(Register.ACQ_COMMAND, 0x03);
+            }
+
+            SystemStatus status = GetStatus();                
+            while(status.HasFlag(SystemStatus.BusyFlag))
+            {
+                status = GetStatus();
+                Thread.Sleep(1);
+            }
+
+            return GetDistanceMeasurement();
+        }
+
+        /// <summary>
+        /// Get the distance measurement.
+        /// </summary>
+        public ushort GetDistanceMeasurement() {
```

Comment:
how is this different from MeasureDistance? Btw both names should be just `GetDistance` - `measurement` is implied

Reply:
I changed all registry retrival to properties, so this is just `Distance`, the current distance measurement in the device.  `MeasureDistance` is higher level, and does several registry operations to get a measurement.